movies/models.py

```diff
@@ -15,6 +17,15 @@ class Movie(models.Model):
     length_min = models.IntegerField(verbose_name='Length')
     trailer_url = models.URLField(max_length=200, null=True, blank=True)
     is_scheduled = models.BooleanField(default=False)
+    time_scheduled_at = models.TimeField(auto_now_add=False, auto_now=False, null=True)
+    date_scheduled_at = models.DateField(auto_now_add=False, auto_now=False, null=True)
+    hall_is_playing = models.ForeignKey(Hall, null=True, on_delete=models.SET_NULL)
```

Comment:
astea 3 field-uri nu merg prea bine aici..
pentru `hall_is_playing`, de exemplu, ar trebui schimbat pk-ul de fiecare data cand un film ruleaza in alta sala which is not ok, iar daca vrem ca un film sa ruleze simultan in sali diferite / cinematografe diferite, nu putem.

Poti face un nou model PlayingTime sau ceva de genul si sa le adaugi acolo (te poti inspira si de la Robert: https://github.com/PurtucRobert/cinemy/blob/79457b46aee34c6f02ce6e4c60d3ddbba852cea8/cinemy/cinema/models.py#L67) si asa poti gestiona easily playing times pentru filme, iti ramane si un istoric al redarilor. Poti cauta "Database normalization" ca sa citesti mai multe sugestii de structurare a tabelelor.

`time_scheduled_at` si `date_scheduled_at` le poti si uni prin DateTimeField, daca vrei